lib/index.js

```diff
@@ -112,7 +112,7 @@ function getBackgroundImageSize(decl, existingBackgroundSize, detectImageSize, r
           prop: 'background-size',
           value: `${size.width}px ${size.height}px`,
         });
-      });
+      }).catch(() => {});
```

Comment:
👍 

Do we have any useful information in the `Error` that could perhaps be logged out to the console?